doc/depthFirstSearch/pgr_depthFirstSearch.rst

```diff
@@ -37,22 +36,20 @@ Undirected DFS algorithm implemented by Boost.Graph.
 Description
 -------------------------------------------------------------------------------
 
-Depth First Search algorithm is a well known traversal algorithm which starts
-from a root vertex (``Root vid``) and visits all the nodes in a graph in the
-depth-first search traversal order. An optional non-negative maximum depth
-parameter (``max_depth``) can be specified to get the results up to a particular
-depth.
+Depth First Search algorithm is a traversal algorithm which starts from a root vertex
+and visits all the vertices in a graph, in the depth-first search traversal order.
 
 **The main Characteristics are:**
 
 - The implementation works for both **directed** and **undirected** graphs.
 - Provides the Depth First Search traversal order from a root vertex to
   a particular maximum depth level.
+- An optional non-negative maximum depth parameter (``max_depth``) can be
+  specified to get the results up to a particular depth.
```

Comment:
An optional non-negative maximum depth parameter to limit the results up to a particular depth.